packages/blockchain-wallet-v4-frontend/src/utils/helpers.ts

```diff
@@ -74,3 +76,31 @@ export const toBase64 = (file: File): Promise<string> => {
     reader.onerror = (error) => reject(error)
   })
 }
+
+/*
+  Analytics usage without redux store coupling.
+*/
+export const analyticsTrackingNoStore = async (data) => {
```

Comment:
Not quite sure we want this over here and also if we want this just for the issue I'm fixing, if that's the case then I would encapsulate all of the implementation configuration into this file also to avoid untidy code on the `index.js`